README.md

```diff
@@ -207,7 +220,45 @@ service:
     curl -XPOST localhost:9080/reindexing/fedora/path -H"Content-Type: application/json" \
         -d '["broker:queue:solr.reindex","broker:queue:fixity","broker:queue:triplestore.reindex"]'
 
-## Troubleshooting
+#### Properties
+| Name      | Description| Default Value |
+| :---      | :---| :----   |
+| reindexing.enabled | Enables/disables the reindexing component. Enabled by default | true | 
+| reindexing.error.maxRedeliveries | Maximum redelivery attempts | 10 | 
+| reindexing.stream | Reindexing jms message stream | broker:queue:reindexing | 
+| reindexing.rest.host | Reindexing service host | localhost | 
+| reindexing.rest.port | Reindexing service port | 9080 |
+| reindexing.rest.prefix | Reindexing rest URI prefix | /reindexing | 
+
+### ActiveMQ Service
+
+This implements a connector to an ActiveMQ broker.
+
+#### Properties
+| Name      | Description| Default Value |
+| :---      | :---| :----   |
+| jms.brokerUrl | JMS Broker endpoint | tcp://localhost:61616 |
+| jms.username | JMS username | null |
+| jms.password | JMS password | null |
+| jms.connections | The JMS connection count | 10 |
+| jms.consumers | The JMS consumer count | 1 |
+
+### Fixity Checking Service
+
+This application can be used in conjunction with the Repository
+Re-Indexer to verify the checksums for all Binary resources in
+the repository.
+
+#### Properties
+| Name      | Description| Default Value |
+| :---      | :---| :----   |
+| fixity.enabled | Enables/disables fixity service  | false |
+| fixity.input.stream | Fixity Service jms  message stream | broker:queue:fixity |
+| fixity.delay | A delay in milliseconds between each fixity check to reduce load on server | 0 |
```

Comment:
Specify, that this is to reduce load on the Fedora server?

Reply:
ah okay.